src/main/kotlin/org/rust/lang/core/psi/ext/RsConstant.kt

```diff
@@ -56,4 +56,6 @@ abstract class RsConstantImplMixin : RsStubbedNamedElementImpl<RsConstantStub>,
     override fun getContext(): PsiElement? = RsExpandedElement.getContextImpl(this)
 
     override fun getUseScope(): SearchScope = RsPsiImplUtil.getDeclarationUseScope(this) ?: super.getUseScope()
+
+    override fun getNameIdentifier(): PsiElement? = identifier ?: underscore
```

Comment:
@Undin actually, I've copied it from `RsAlias.kt`, but looks like we don't have `DuplicateItemError` for use items, so that's why it works in case of `RsAlias`. So probably I should get rid of this override and use `identifier ?: underscore` instead of `getNameIdentifier()` everywhere in the code?